src/systems/filecoin_blockchain/storage_power_consensus/storage_power_actor_state.go

```diff
@@ -127,6 +135,10 @@ func (st *StoragePowerActorState_I) _selectMinersToSurprise(challengeCount int,
 	return selectedMiners
 }
 
+func (st *StoragePowerActorState_I) _shouldChallenge(rt Runtime, networkPower block.StoragePower) bool {
+	panic("TODO")
+}
+
```

Comment:
Why all the additions in this file? Some of these are defined in `ChallengeStatus.go` (no reason you should have known that and we should probably split out interfaces which have corresponding go files). I assume it was left in there by @zixuanzh since it is technically "actor code."

Reply:
Because it didn't compile without them